src/types/layout.ts

```diff
@@ -0,0 +1,5 @@
+export type LayoutType = {
```

Comment:
It's good we have a type for this now, but why suffix it with `Type`, why not just `Layout`?